backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExport.kt

```diff
@@ -183,23 +182,31 @@ internal class ObjCExport(val context: Context, symbolTable: SymbolTable) {
 
         """.trimIndent())
 
-
+        val uiDeviceFamilyValues = if (target.isTvOs) {
+            """
+                |<integer>3</integer>
+            """.trimIndent()
+        } else {
+            """
+                |<integer>1</integer>
+                |<integer>2</integer>
+            """.trimIndent()
+        }
         contents.append(when (target.family) {
             Family.IOS -> """
                 |    <key>MinimumOSVersion</key>
                 |    <string>$minimumOsVersion</string>
                 |    <key>UIDeviceFamily</key>
                 |    <array>
-                |        <integer>1</integer>
-                |        <integer>2</integer>
+                |       $uiDeviceFamilyValues
                 |    </array>
 
                 """.trimMargin()
             Family.OSX -> ""
             else -> error(target)
         })
 
-        if (target == KonanTarget.IOS_ARM64) {
+        if (target.architecture == Architecture.ARM64) {
```

Comment:
Does Xcode emit `UIRequiredDeviceCapabilities` with `arm64` for tvOS arm64 device framework?

Reply:
Nope, it's incorrect. 👍